.config/rollup.config.js

```diff
@@ -32,11 +32,45 @@ export default {
              * non-builtin CommonJS modules in Node correctly.
              */
             interop: 'defaultOnly',
+            /**
+             * This is required to prevent the error:
+             *
+             * TypeError: createContext only works in Client Components.
+             * Add the "use client" directive at the top of the file to use it.
+             *
+             * -----
+             *
+             * Here is the Rollup documentation on `banner`:
+             * A string to prepend/append to the bundle. You can also
+             * supply a function that returns a Promise that resolves
+             * to a string to generate it asynchronously
+             *
+             * (Note: banner and footer options will not
+             * break sourcemaps).
+             */
+            banner: "'use client';",
         },
         {
             dir: 'dist',
             format: 'cjs',
             exports: 'named',
+            /**
+             * This is required to prevent the error:
+             *
+             * TypeError: createContext only works in Client Components.
+             * Add the "use client" directive at the top of the file to use it.
+             *
+             * -----
+             *
+             * Here is the Rollup documentation on `banner`:
+             * A string to prepend/append to the bundle. You can also
+             * supply a function that returns a Promise that resolves
+             * to a string to generate it asynchronously
+             *
+             * (Note: banner and footer options will not
+             * break sourcemaps).
+             */
```

Comment:
I think this comment can be removed since we have already have the comment above for the ESM build.

Reply:
Gotcha, I'll update it. Should I delete it entirely or say something like "see above comment"?

Reply:
Either is fine 🙂

Reply:
Done!